Codes/insert.php

```diff
@@ -65,17 +65,60 @@
 
             //Updating grade window
             $c=count($_POST['grade']);
-            $i=1;
-            for($i=0;$i<$c;$i++)
+            $i=0;
+            $r=$_POST['grade'][$i];
+            if($i<$c && $_POST['lb'][$i]<=$_POST['ub'][$i] && $_POST['ub'][$i]==100)
             {
-                $r=$_POST['grade'][$i];
                 $sql="UPDATE gradewindow SET lower_cutoff=".$_POST['lb'][$i].", upper_cutoff=".$_POST['ub'][$i]." WHERE course_code='$cd' AND grade='$r'";
 
+                if ($conn->query($sql) !== TRUE)
+                {
+                    header('Location: edit.php?course='.$cd.'&error=ERROR UPDATING GRADE CUT OFF');
+                    exit();
+                }
+            }
+            else if($i<$c)
+            {
```

Comment:
please try to in cooperate first and last case inside for loop only. It will significantly decrease the lines of code

for(conditon)
{ 
    if(first case) // include error part inside this if construct only (i.e. inside this if only you should check if error occurred while updating)
    {}
    else if(lase case) //  include error part inside this else if construct only (i.e. inside this if only you should check if error occurred while updating)
     {}
     else // rest case
     {}
}